### PR TITLE
Refactor MenuBar.qml

### PR DESCRIPTION
**Pull request by DongTrieu1998**
Change Row to RowLayout.

### CODE REVIEW

**Comment by DongTrieu1998** [comment]
@thanhdat369 I change Row to RowLayout. Please take a look. Thank you.

File: fudy/MenuBar.qml [hunk 1]
```diff
@@ -27,42 +27,50 @@ Rectangle {
...
 		}
 		Image {
-			width: 30
-			height: 30
+			Layout.preferredWidth: 30
+			Layout.preferredHeight: 30
+			fillMode: Image.PreserveAspectCrop
 			source: "qrc:/image/settings.svg"
 		}
 		Image {
```
Comment (ghost) [inline fudy/MenuBar.qml:70]: Please create this Image to the new Component (maybe in the future you will create custom MenuBarItem)

Comment (DongTrieu1998) [reply fudy/MenuBar.qml:70]: Can you explain what you mean, please?

**Review by ghost — APPROVED** [review]
It's work. However please update following the comment above.

File: fudy/MenuBar.qml (hunk 1 above)
Comment (ghost) [reply fudy/MenuBar.qml:70]:
```
\\ MenuBarItem.qml
Item {
        id: root
        property string sourceUrl: ""
        Layout.preferredWidth: 30
	Layout.preferredHeight: 30
        Image {
			        fillMode: Image.PreserveAspectCrop
			        source: root.sourceUrl
        }
}
\\ other.qml
MenuBarItem {
     sourceUrl: "qrc:/image/settings.svg"
}
MenuBarItem {
     sourceUrl: "qrc:/image/home.svg"
}
```

Comment (DongTrieu1998) [reply fudy/MenuBar.qml:70]: got it, thank you

**Pull request merged**